### PR TITLE
add ignore check spelling for music format

### PR DESCRIPTION
**Pull request by tkhmy**
Signed-off-by: tkhmy <tkh.my.p@gmail.com>

- Add ignore the music file: *.wav, *.mp3, *.m4a

### CODE REVIEW

File: cspell/.cspell.json [hunk 1]
```diff
@@ -46,11 +46,16 @@
         "**/*.caffemodel",
         "**/*.cu",
         "**/*.html",
+        "**/*.jpeg",
+        "**/*.jpg",
+        "**/*.mp3",
```
Comment (kmiya) [inline cspell/.cspell.json:51]:
```suggestion
        "**/*.m4a",
        "**/*.mp3",
```
cSpell should ignore `jpeg` and `jpg` by default. So I'd like to delete the lines.
Instead, ignoring `m4a` might be helpful.

File: cspell/.cspell.json [hunk 2]
```diff
@@ -46,11 +46,16 @@
...
         "**/*.onnx",
         "**/*.osm",
         "**/*.pcd",
         "**/*.pt",
         "**/*.stl",
+        "**/*.svg",
```
Comment (kmiya) [inline cspell/.cspell.json:57]:
```suggestion
```
cSpell can detect typos in SVG; it is really helpful.
Please consider adding ignore patterns instead.
<https://github.com/tier4/autoware-spell-check-dict/blob/main/cspell/.cspell.json#L36-L42>

**Review by kmiya — CHANGES_REQUESTED** [review]

File: cspell/.cspell.json (hunk 1 above)
Comment (tkhmy) [reply cspell/.cspell.json]: Okay, thanks for info!

**Comment by kmiya** [comment]
Thanks!

**Pull request merged**